src/Component/Nav/Nav.js

```diff
@@ -0,0 +1,136 @@
+                        <h2 className="item">
+                          <span role="menuitem" tabIndex="0" className="link">
+                            <span
+                              className="arrow-wrapper"
+                              onClick={this.handleProductNav}
+                            >
+                              상품보기
+                            </span>
+                          </span>
+                        </h2>
+                      </li>
+                      <li className="items-inserted">
+                        <h2 className="item">
+                          <span
+                            role="link"
+                            tabIndex="0"
+                            className="link-limited-purchase"
+                          >
+                            <Link to="/bulkpackageSale">대용량 팩 할인</Link>
+                            <span className="limited-purchase">최대 20%</span>
+                          </span>
+                        </h2>
+                      </li>
+                      <li className="items">
+                        <h2 className="item">
+                          <span role="menuitem" tabIndex="0" className="link">
+                            <span className="arrow-wrapper">
+                              론위즐리 이야기
+                            </span>
+                          </span>
+                        </h2>
+                      </li>
+                      <li className="items">
+                        <h2 className="item">
+                          <span role="menuitem" tabIndex="0" className="link">
+                            <span className="arrow-wrapper">고객센터</span>
+                          </span>
+                        </h2>
+                      </li>
+                    </ul>
+                  </div>
+                </nav>
+                <div className="user-info-wrapper">
+                  <span role="link" className="login">
+                    <Link to="/login">로그인</Link>
+                  </span>
+                  <div className="basket-wrapper">
+                    <span className="basket">장바구니</span>
+                    <div className="basket-quantity-wrapper">
+                      <span className="quantity-count">0</span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </header>
+          </div>
+
+          <div
+            className={
+              this.state.subNavActive
+                ? "NabSubContainer"
+                : "NabSubContainer-none"
+            }
+          >
+            {this.state.subNavActive ? (
+              <NavSub handleProductNav={this.handleProductNav} />
+            ) : null}
+          </div>
+        </div>
+
+        {this.props.activeProductNav ? (
+          <ProductNav
+            changeSideBarValidInProduct={this.props.changeSideBarValidInProduct}
+          />
+        ) : null}
```

Comment:
```suggestion
        {this.props.activeProductNav && (
          <ProductNav
            changeSideBarValidInProduct={this.props.changeSideBarValidInProduct}
          />
        )}
```
true case만 이용할 때는 && 연산자 사용해주세요

Reply:
수정완료했습니다.